Sources/AST/ASTPass/ASTPassContext.swift

```diff
@@ -267,3 +273,7 @@ private struct IsAssignment: PassContextEntry {
 private struct IsPropertyDefaultAssignment: PassContextEntry {
   typealias Value = Bool
 }
+
+private struct IsFunctionCallArgumentLabelContext: PassContextEntry {
```

Comment:
I think we should drop the `Context` here to be consistent with the naming above